src/plugins/shipping-rates/queries/flatRateFulfillmentMethod.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @name flatRateFulfillmentMethod
+ * @method
+ * @memberof Fulfillment/Queries
+ * @summary Query the Shipping collection for a single of fulfillment method
```

Comment:
grammar error, I believe you meant, "Query the Shipping collection for a single fulfillment method"